account_invoice_constraint_chronology/model/account_move.py

```diff
@@ -95,6 +100,8 @@ def _conflicting_inv_after_sequence_before_inv_date_domain(self):
         return expression.AND(
             [
                 (
+                    ("journal_id", "=", self.journal_id.id),
+                    ("move_type", ">", self.move_type),
```

Comment:
I don't understand the ">" operator in the move_type comparisons... ¿?

Reply:
wow that was a typo indeed.

Reply:
Ok, we must take into account that if the journal has the "different sequence" checkbox marked for refunds (refund_sequence), then it's true that the move_type should be checked. However, if not, the situation changes... there could be sequence conflicts among different types (https://github.com/OCA/account-financial-tools/blob/16.0/account_invoice_constraint_chronology/model/account_move.py#L14). 
In my opinion, it would be best to unify the domain methods to avoid confusion and duplicate code.
What do you all think?